HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,124 @@
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+ggplot(diamonds, aes(x = cut, fill = clarity)) +
+  geom_bar(position = "dodge") +
+  labs(title = "My Diamond Collection", x = "Diamond Cut", y = "Number of Diamonds", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = 0.3) +
+  annotate("text", x = 3.9, y = 4500, label = "My Best Diamonds, \nof course") +
+  theme(plot.title = element_text(hjust = 0.5))
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+ggplot(iris, aes(x = Sepal.Length, y = Petal.Length, shape = Species, color = Species)) +
+  geom_point() +
+  geom_smooth(method = "lm", color = "black", se = FALSE) +
+  facet_wrap(. ~ Species, scales = "free_y") 
+#I do not know how to switch the graphs around in the correct order, if I have time I'll go back and play with this one more later
+```
+
+
+### Graph 3
+You'll need the information in this first box to create the graph
+```{r, echo = TRUE}
+data("mpg")
+corvette <- mpg[mpg$model == "corvette",]
+#install
+require("ggrepel") #useful for making text annotations better, hint hint
+set.seed(42)
+```
+
+Now using the mpg dataset and the corvette dataset, make this graph:
+
+```{r graph 3 code}
+ggplot() +
+  geom_point(data = mpg, mapping = aes(x = displ, y = hwy)) +
+  geom_point(data = corvette, mapping = aes(x = displ, y = hwy), color = "blue") +
+  geom_text_repel(data = corvette, aes(x = displ, y = hwy, label= paste("Corvette,",year))) +
+  labs(title = "Corvettes are a bit of an outlier") +
+  scale_x_continuous(limits = c(1,8), breaks = c(1:8))
+```
+
+There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
+
+
+### Graph 4 
+```{r, echo = T}
+data(mpg)
+
+#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
+library(RColorBrewer)
+display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
+```
+
+The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
+
+
+
+Now using the above mpg dataset, make this graph
+
+```{r graph 4 code}
+ggplot(data = mpg, aes(x = cty, y = class)) +
+  geom_boxplot(orientation = "y", outlier.shape = NA) + 
+  geom_jitter(shape = 16, position = position_jitter(0), aes(color = class), alpha = 0.6) +
+  scale_x_continuous(limits = c(10,35), breaks = seq(10, 35, by = 5)) +
+  scale_color_brewer(palette = "Set2") +
+  labs(title = "Horizontal BoxPlot of City MPG and Car Class", x = "Car Class", y = "City mpg") +
+  theme(panel.border = element_blank(), panel.grid = element_blank()) + #removed grid and borders
```

Comment:
I had some trouble with this part. I swear I wrote these same lines for the theme() function as you, but my graph kept looking the same, and I thought I was doing it the wrong way, but it worked for you... and know that I'm retrying in my file, it works! I don't understand what happened, but peer-reviewing is a good thing.